Tests/Unit/PollerTests.swift

```diff
@@ -0,0 +1,85 @@
+import Quick
+import Nimble
+@testable import RRemoteConfig
+
+class PollerSpec: QuickSpec {
+    override func spec() {
+        describe("init function") {
```

Comment:
I kind of wonder whether the whole `init` test block should be here. The `delay` and `timer` variables shouldn't be accessed outside of their class, and these tests don't really guarantee correct functionality because it doesn't test how they're used within the class.

I'm just not sure how you would test that the default is 1 hour. Maybe there's a way to mock `DispatchQueue`? What do you think?

Reply:
I can probably inject the queue as an init dependency (with main queue as the default) and then inject a mock in the test.

Reply:
Mocking the queue doesn't help because the async function just receives an opaque void closure.
Instead I tried to mock the RunLoop but the init of the mock instance seems to create a new runloop which causes weird crashes :(

Reply:
I used a protocol for the runloop param and I think that works and is now injectable and mockable!